examples/widgets/canvas/lv_example_canvas_10.c

```diff
@@ -6,9 +6,8 @@
 
 static void timer_cb(lv_timer_t * timer)
 {
-    static int16_t counter = 0;
-    const char * string = "windstorrrrrrrrrrrrrrrrm~>>>";
-    const int16_t string_len = (int16_t)lv_strlen(string);
+    const char * string = "lol~ I'm wavvvvvvving~>>>";
+    const int16_t string_len = lv_strlen(string);
```

Comment:
The CI is failing because this variable definition was removed and is now undeclared when used below. I verified locally that adding it back will fix the CI.

```suggestion
    static int16_t counter = 0;
    const char * string = "lol~ I'm wavvvvvvving~>>>";
    const int16_t string_len = lv_strlen(string);
```

Reply:
Ahh, thanks!